src/utils/vc.js

```diff
@@ -171,18 +164,11 @@ export async function verifyCredential(credential, resolver = null, compactProof
 /**
  * Check that credential is verified, i.e. the credential has VCDM compliant structure and the `proof`
  * (signature by issuer) is correct.
- * @param {object} credential - verifiable credential to be verified.
- * @param {DIDResolver} resolver - Resolver for DIDs.
- * @param {Boolean} [compactProof] - Whether to compact the JSON-LD or not.
- * @param {Boolean} [forceRevocationCheck] - Whether to force revocation check or not.
- * Warning, setting forceRevocationCheck to false can allow false positives when verifying revocable credentials.
- * @param {object} [revocationAPI] - An object representing a map. "revocation type -> revocation API". The API is used to check
- * revocation status. For now, the object specifies the type as key and the value as the API, but the structure can change
- * as we support more APIs there are more details associated with each API. Only Dock is supported as of now.
+ * @param {object} [params] Verify parameters (TODO: add type info for this object)
```

Comment:
Same comment as above, credential is the primary argument and it should be outside the object.
Why can't the same comment bodies be kept for the params object?

Reply:
same reply as above